.gitignore

```diff
@@ -36,6 +36,7 @@ test/lit/lit.site.cfg.py
 
 # files commonly related to building out-of-tree
 /build/
+/embuild/
```

Comment:
Is this a standard name emitted by some tool?

Reply:
That's something I made up, as I typically have to compile both wasm-opt and binaryen.js to run the tests respectively update the tests, so I am using two folders for the different builds, and `embuild` seemed appropriate for the Emscripten one :)

Reply:
Removed this one again, as I could ofc just use two subfolders if I weren't so lazy heh

Reply:
Thanks. I don't see any reason not to add this to the .gitignore since we already have `/build/` in there, which I think is in a similar situation.